sktime/forecasting/compose/_pipeline.py

```diff
@@ -93,7 +93,11 @@ def _check_steps(self, estimators, allow_postproc=False):
         COERCIBLE_SCITYPES = set(COERCIBLE_SCITYPES) - set(ALLOWED_SCITYPES)
         ACCEPTED_SCITYPES = ALLOWED_SCITYPES + list(COERCIBLE_SCITYPES)
 
-        est_scitypes = [scitype(x) for x in estimators]
+        est_scitypes = []
+        for x in estimators:
+            # Check if it's a forecaster 
```

Comment:
I do not think this is a good fix - it will only result in `["forecaster"]` and ignore the types of the other estimators.